### DISCUSSION TITLE
寻找比微信群更开放，比 Github Discussions 更实时更活跃的交流平台

### DISCUSSION THREAD

**Discussion by yangmingshan** [vue-mini/vue-mini#71] [General]
如题。虽然我们现在有微信交流群，以及这个 Discussions 论坛，但我还是觉得他们各有各的问题。微信群实时性很强，活跃度也很高，但是太封闭了，没办法公开邀请人。Github Discussions 很开放，但是实时性不足，而且根据我的经验来看活跃度也不会很高，大伙儿还是更喜欢群聊。国外社区一般都用 [Discord](https://discord.com) 这个平台，但是我不知道这个在国内会不会太冷门，而且它需要翻墙，有点使用门槛。或者国内有什么比较好的 Discord 替代品吗？

大伙儿帮忙支支招，一起讨论一下。

**Comment by w2xi** [comment]
readme 贴个 作者和微信群的二维码咋样？就很方便拉人了，然后还可以搞个赞助码。

比如：

## 社区

作者二维码 | 微信社区群二维码

## 赞助

赞助码

**Reply by yangmingshan** [reply]
微信群超过 200 人，就只能通过邀请入群了，没有二维码。所以我说微信群封闭。

**Comment by wzs28150** [comment]
kook了解一下https://m.baidu.com/video/page?pd=video_page&nid=11763895615643578293&sign=11096901723880006431&word=kook%E6%98%AF%E4%BB%80%E4%B9%88%E8%BD%AF%E4%BB%B6&oword=kook%E6%98%AF%E4%BB%80%E4%B9%88%E8%BD%AF%E4%BB%B6&atn=index&frsrcid=48304&ext=%7B%22jsy%22%3A1%7D&top=%7B%22sfhs%22%3A1%2C%22_hold%22%3A2%7D&fr0=A&fr1=A&ms=1&lid=16580520167781410523&referlid=16580520167781410523&frorder=10&_t=1721042191173

**Reply by yangmingshan** [reply]
这个感觉也很冷门啊，我都没听过 😂

**Reply by wzs28150** [reply]
据说是国内版的Discord😌

**Reply by zerob13** [reply]
这个就是原来的 开黑啦， 打游戏的时候用的挺多的

**Comment by mhpsy** [comment]
我是认为论坛的形式会比微信群更加有“营养”，微信群里面什么话都有，摸鱼的吹牛逼的。如果什么都看的话会浪费很多时间，而且一般也收获不到什么有价值的东西。
而论坛会根据不同的话题，分割成为不同的部分，只需要看自己需要的部分就好了。
但是没有活跃度都是白扯，还是要有人发言才比较好。

**Reply by yangmingshan** [reply]
是的，现在社区还处于很早期的阶段，所以还是想让大家更积极的参与，活跃起来

**Comment by Loongphy** [comment]
类 Discord 的话，国内一般是用于游戏开黑语音，比如 kook，oopz。虽然很活跃，但用于开发社区讨论我保持质疑。至于飞书钉钉之流，老实说，我假期不会打开看，很难办~

**Comment by yangmingshan** [comment]
QQ 群呢？你们还用 QQ 吗？

**Reply by calandnong** [reply]
会用，经常

**Reply by Loongphy** [reply]
QQ 有个频道功能，类 Discord

**Reply by GuoJikun** [reply]
qq 群更方便一些，而且功能比微信强大

**Reply by Songyakang** [reply]
qq群确实方便

**Comment by calandnong** [comment]
对于官方讨论平台这块，据我了解国内用discord的还是比较少数的，特别是小程序开发的用户，但是这个开发者群体其实更多的讨论是qq群/小程序官方社区，如果是选择国内平台的话，或许可以试试：[https://www.imdodo.com/，或者自己构建一个社区平台，或者通过一些方式引流到GitHub](https://www.imdodo.com/%EF%BC%8C%E6%88%96%E8%80%85%E8%87%AA%E5%B7%B1%E6%9E%84%E5%BB%BA%E4%B8%80%E4%B8%AA%E7%A4%BE%E5%8C%BA%E5%B9%B3%E5%8F%B0%EF%BC%8C%E6%88%96%E8%80%85%E9%80%9A%E8%BF%87%E4%B8%80%E4%BA%9B%E6%96%B9%E5%BC%8F%E5%BC%95%E6%B5%81%E5%88%B0GitHub) Discussions

**Reply by calandnong** [reply]
这里的话，感觉前期还是建设氛围比较重要，让更多人参与进来，或者有一个需求做个开源小程序出来？把核心的人群把握活跃起来先，例如https://github.com/elk-zone/elk？

**Comment by yangmingshan** [comment]
https://vuemini.org/guide/community.html
